src/lib/es2015.core.d.ts

```diff
@@ -487,11 +487,13 @@ interface StringConstructor {
     fromCodePoint(...codePoints: number[]): string;
 
     /**
-     * String.raw is intended for use as a tag function of a Tagged Template String. When called
-     * as such the first argument will be a well formed template call site object and the rest
-     * parameter will contain the substitution values.
+     * String.raw is usually used as a tag function of a Tagged Template String. When called as
+     * such, the first argument will be a well formed template call site object and the rest
+     * parameter will contain the substitution values. It can also be called directly, for example,
+     * to interleave strings and values from your own tag function, and in this case the only thing
+     * it needs from the first argument is the raw property.
      * @param template A well-formed template string call site representation.
      * @param substitutions A set of substitution values.
      */
-    raw(template: TemplateStringsArray, ...substitutions: any[]): string;
+    raw(template: { raw: readonly string[] | ArrayLike<string>}, ...substitutions: any[]): string;

```

Comment:
I wouldn't necessarily include `ArrayLike` - I'd keep it simple here personally

Reply:
@DanielRosenwasser: I agree that it would be better (like many other places), but I followed what the language is doing

```js
> String.raw({raw: {1: "y", 0: "x", length: 2}}, [3])
"x3y"
```